sway/ipc-server.c

```diff
@@ -125,6 +125,8 @@ struct sockaddr_un *ipc_user_sockaddr(void) {
 	return ipc_sockaddr;
 }
 
+// FreeBSD supports getting uid/gid, but not pid
```

Comment:
Is there something else we could use to get the pid of the client? Have you filed a bug with FreeBSD asking for this feature? This is crucial to IPC security.